src/D2L.Bmx/Browser.cs

```diff
@@ -0,0 +1,64 @@
+using PuppeteerSharp;
+
+namespace D2L.Bmx;
+
+public class Browser {
```

Comment:
nit: make it clear this is for utility stuff with no internal state

```suggestion
public static class Browser {
```